pcsx2/gui/MainFrame.cpp

```diff
@@ -465,13 +465,13 @@ MainEmuFrame::MainEmuFrame(wxWindow* parent, const wxString& title)
 	m_menuCDVD.Append( GetPluginMenuId_Settings(PluginId_CDVD), _("Plugin &Menu"), m_PluginMenuPacks[PluginId_CDVD] );
 
 	m_menuCDVD.AppendSeparator();
-	m_menuCDVD.Append( MenuId_Src_Iso,		_("&Iso"),		_("Makes the specified ISO image the CDVD source."), wxITEM_RADIO );
```

Comment:
*sigh* I missed the instance of ``Iso`` in line [464](https://github.com/PCSX2/pcsx2/pull/1651/files#diff-1d6e5cd2c2b8d912c67d623d8e73d31dR464).